style.css

```diff
@@ -0,0 +1,33 @@
+#okvir{
+    width: 500px;
+    height: 500px;
+    border: solid black 4px;
+    position: relative;
+    margin: 0 auto;
+    display: flex;
+}
+
+#kocka{
+    width: 50px;
+    height: 50px;
+    position: absolute;
```

Comment:
Ako imaš više elemenata koji dele isti css onda napiši to prvo, pa onda ono po čemu se razlikuju.

Reply:
Ne razumem najbolje? Width, height i position su mi u svakom elementu na prvim pozicijama?

Reply:
```
#kocka, 
#kocka-zelena, 
#kocka-crvena{
    width: 50px;
    height: 50px;
    position: absolute;
}
```
I onda izbrišeš dole kod koji se ponavlja